### PR TITLE
Fix code typos

### PR DESCRIPTION
**Pull request by wavexx**
This fixes a few misspellings in code.

Also addresses https://github.com/prusa3d/Prusa-Firmware/pull/3846#discussion_r1053439879

### CODE REVIEW

File: Firmware/xyzcal.cpp [hunk 1]
```diff
@@ -135,9 +135,9 @@ pos_mm_t pos_2_mm(float pos){
 	return pos * 0.01f;
 }
 
-void xyzcal_meassure_center(void)
+void xyzcal_meassure_enter(void)
 {
-	DBG(_n("xyzcal_meassure_center\n"));
+	DBG(_n("xyzcal_meassure_enter\n"));
```
Comment (Panayiotis-git) [inline Firmware/xyzcal.cpp]: Is it meassure or measure?

Comment (wavexx) [reply Firmware/xyzcal.cpp]: It's measure. Isn't it fixed in the next commit?

Comment (wavexx) [reply Firmware/xyzcal.cpp]: This misspelling bothered me for a looong time ;)

**Review by 3d-gussner — APPROVED** [review]

**Review by leptun — APPROVED** [review]
Approved and I pushed one more typo fix

**Review by DRracer — APPROVED** [review]
:pray:

**Pull request merged**